cocotb/utils.py

```diff
@@ -196,7 +202,16 @@ def unpack(ctypes_obj, string, bytes=None):
         :exc:`ValueError`: If length of *string* and size of *ctypes_obj*
             are not equal.
         :exc:`MemoryError`: If *bytes* is longer than size of *ctypes_obj*.
+
+    .. deprecated:: 1.5
+        Its functionality is now built into Python 3 as ``memoryview(ctypes_obj)[:bytes] = string``.
```

Comment:
```suggestion
        This functionality is now built into Python 3 as ``memoryview(ctypes_obj)[:bytes] = string``.
```
It would also be nice to link this code to https://docs.python.org/dev/library/stdtypes.html#memoryview